packages/lit-html/src/lit-html.ts

```diff
@@ -323,8 +323,26 @@ const tag =
 export const html = tag(HTML_RESULT);
 
 /**
- * Interprets a template literal as an SVG template that can efficiently
+ * Interprets a template literal as an SVG fragment that can efficiently
  * render to and update a container.
+ *
+ * ```ts
+ * const rect = svg`<rect width="10" height="10"></rect>`;
+ *
+ * const myImage = html`
+ *   <svg viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg">
+ *     ${rect}
+ *   </svg>`;
+ * ```
+ *
+ * The `svg` tagged template literal (TTL) should only be used for SVG
+ * fragments, or elements that would be contained within an `<svg>` html tag. A
```

Comment:
Since we have a lot of tags to talk about, maybe use "element" rather than tag for an HTML element.
Also, maybe emphasize the "inside". 

KInd of on the fence whether we should include "HTML" here. "an `<svg>` element" vs "an `<svg>` HTML element" ... The latter is perhaps more precise but harder to read.

```suggestion
 * fragments, or elements that would be contained **inside** an `<svg>` HTML element. A
```